.github/workflows/publish.yaml

```diff
@@ -0,0 +1,33 @@
+name: Publish
```

Comment:
the trusted publisher config  on pypi must reference exactly the name of this file `publish.yaml`

Reply:
Done.